react-native/training/src/app/screens/Home/screens/TodoList/components/Item/styles.js

```diff
@@ -0,0 +1,39 @@
+import { StyleSheet } from 'react-native';
+
+import { gray, newYorkPink } from '../../../../../../../constants/colors';
+
+export default StyleSheet.create({
+  container: {
+    flexDirection: 'row',
+    justifyContent: 'space-between',
+    paddingHorizontal: 14,
+    paddingVertical: 8,
+    borderBottomWidth: 1,
+    borderColor: gray
+  },
+  containerSelected: {
+    backgroundColor: gray
+  },
+  textItem: {
+    flexGrow: 1,
+    flexShrink: 1,
+    flexBasis: 'auto'
+  },
+  containerOptions: {
+    alignItems: 'center',
+    flexDirection: 'row',
+    flexGrow: 0,
+    flexShrink: 0,
+    flexBasis: 'auto',
+    marginLeft: 5
+  },
+  textButton: {
+    color: newYorkPink,
+    fontWeight: '600'
+  },
+  button: {
```

Comment:
Add more margin to this component and do it more large. It's difficult to touch it